.github/ISSUE_TEMPLATE/bug-report.md

```diff
@@ -1,12 +1,3 @@
----
-name: 🦟  Bug report
-about: Report a bug to help us improve.
-title: ''
-labels: 'bug,:reproduce'
-assignees: ''
-
----
-
```

Comment:
^this would break the issue template

Reply:
I don't understand what's happening here, to be honest. I didn't include anything regarding `assignees`, and I definitely checked multiple times before submitting to ensure that everything was correct, so I can't speak to how this breaks anything.

Reply:
I'm not sure what changed on this line. Looks like a whitespace change? @sabrinabuckets this may have been done automatically by your IDE when you saved the file. Can you remove this line change?

Reply:
I did this all in the GitHub web interface, not an IDE. To be honest, I am not entirely sure _how_ to make additional changes. Could you possibly show me as part of our 1-1 later?

Reply:
Ah, my mistake, Mike corrected, no change needed. Thanks!